src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -79,6 +79,14 @@ add_spectre_parallel_executable(
   "EvolveGhCce"
   EvolveGhAndCharacteristic
   Evolution/Executables/GeneralizedHarmonic
-  "EvolutionMetavars<3>"
+  "EvolutionMetavars<3, false>"
+  "${LIBS_TO_LINK};Cce"
+)
+
+add_spectre_parallel_executable(
+  "EvolveCcm"
```

Comment:
"EvolveGhCcm" for consistency?

Reply:
Oh yeah probably a good idea. Missed that.

Reply:
I'd prefer to rename `EvolveGhCce` to `EvolveCce` since CCE/CCM = GH + Characteristic. It's not reasonable to put GH together with either CCE or CCM. I've just added one more fixup commit.

Reply:
I don't like this. We have a standalone CCE executable that doesn't have anything to do with GH. I know it's called `CharacteristicExtract`, but between that and `EvolveCce` I have no clue what the difference is. `EvolveGhCce` and `EvolveGhCcm` make it clear that a GH evolution is happening here in addition to a characteristic evolution.

Reply:
Also there are other formulations of the Einstein equations besides GH...

Reply:
Alright. I've added `Gh` to both executables.